src/main.jsx

```diff
@@ -4,17 +4,55 @@ import './styles/main'
 
 import React from 'react'
 import { render } from 'react-dom'
+import { Router, Route, Redirect, hashHistory } from 'react-router'
 import { I18n } from './plugins/preact-polyglot'
 
-import App from './components/app'
+import App from './components/App'
+import Table from './components/Table'
 
 const context = window.context
 const lang = document.documentElement.getAttribute('lang') || 'en'
 
 document.addEventListener('DOMContentLoaded', () => {
   render((
     <I18n context={context} lang={lang}>
-      <App />
+      <Router history={hashHistory}>
+        <Route component={(props) =>
+          <App {...props} />}
+        >
+          <Redirect from='/' to='files' />
+          <Route
+            path='files'
+            component={(props) =>
+              <Table {...props} />
```

Comment:
As long as the `App` children is always a `Table` component, is it really relevant to externalize it in the _main.jsx_ file instead of letting it live in the _app.jsx_ file directly?